.travis.yml

```diff
@@ -5,7 +5,6 @@ node_js:
   - '8'
   - '6'
 env:
-  - REACT_VERSION=0.14.9
   - REACT_VERSION=15.0
```

Comment:
Can we make this `15.3` to be explicit please?

Reply:
Done!